packages/terra-clinical-label-value-view/src/terra-dev-site/doc/example/LabelValueViewTextTermAndDef.jsx

```diff
@@ -10,7 +10,8 @@ const LabelValueViewText = () => (
         }
       `}
     </style>
-    <dl id="dl-wrapper">
+    <h4 id="topicText">Vitals Topic Text</h4>
```

Comment:
We can remove "Topic Text" as that's redundant.
```suggestion
    <h4 id="topicText">Vitals</h4>
```

Reply:
Done!